recipe/meta.yaml

```
@@ -20,16 +20,16 @@ requirements:
     - python >=3.6
   run:
     - python >=3.7
-    - packaging >=17.0
-    - numpy  >=1.17.2
-    - pyyaml  >=5.4
-    - pytorch >=1.9
-    - tqdm  >=4.57.0
-    - tensorboard >=2.9.1
-    - fsspec[http] >=2021.05.0, !=2021.06.0
-    - torchmetrics >=0.7.0
-    - pyDeprecate >=0.3.1
-    - typing-extensions >=4.0.0
+    - packaging >=17.0, <=21.3
+    - numpy >=1.17.2, <1.23.1
+    - pyyaml >=5.4, <=6.0
+    - pytorch >=1.9.*, <1.13.0
+    - tqdm >=4.57.0, <4.65.0
+    - tensorboard >=2.9.1, <2.11.0
+    - fsspec[http] >2021.06.0, <2022.8.0
+    - torchmetrics >=0.7.0, <0.10.1
+    - typing-extensions >=4.0.0, <=4.4.0
+    - lightning-utilities >=0.3.0, <0.4.0
```

there shall not be any upper bounds
```suggestion
    - packaging >=17.0
    - numpy >=1.17.2
    - pyyaml >=5.4
    - pytorch >=1.9.0
    - tqdm >=4.57.0
    - tensorboard >=2.9.1
    - fsspec[http] >2021.06.0
    - torchmetrics >=0.7.0
    - typing-extensions >=4.0.0
    - lightning-utilities >=0.3.0, !=0.4.0
```